jenkins-integration/jenkins-jobs/common/scripts/job-steps/020_install_pe.sh

```diff
@@ -21,9 +21,11 @@ bundle exec beaker \
         --log-level debug \
         --no-color \
         --tests \
-jenkins-jobs/common/scripts/ops-deployment/hack_hostname_into_etc_hosts.rb,\
-jenkins-jobs/common/scripts/ops-deployment/disable_firewall.rb,\
-beaker/install/pe/10_install_pe.rb
+beaker/install/shared/hack_hostname_into_etc_hosts.rb,\
+beaker/install/shared/disable_firewall.rb,\
+beaker/install/pe/10_install_pe.rb,\
+beaker/install/shared/configure_permissive_server_auth.rb,\
+beaker/install/pe/99_restart_server.rb
```

Comment:
Sure, that's probably a good suggestion.  For now I don't have any plans for this to support any SUT OS's besides cent7 but we can definitely improve / broaden that in the future.